spec/rubocop/cop/lint/circular_argument_reference_spec.rb

```diff
@@ -68,7 +68,37 @@
       end
     end
 
-    context 'when there are multiple offense keyword arguments' do
+    context 'when the keyword argument is not circular, but calls the same
```

Comment:
It's best to split a multiline string in two with `\`, otherwise you'd be inserting whitespaces on the second line.